### ISSUE TITLE
utils\spacy_utils.py", line 109, in ensure proper_language_model if nlp.path is none: AttributeError:'English' Object has no attribute 'path'

### ISSUE THREAD

**Issue by seeni18** [RasaHQ/rasa_nlu#559]
**rasa NLU version** (e.g. `0.7.3`):

**Used backend / pipeline** (`mitie`, `spacy_sklearn`, ...):Spacy_sklearn

**Operating system** (windows, osx, ...): Windows 10

**Issue**:utils\spacy_utils.py", line 109, in ensure proper_language_model if nlp.path is none: AttributeError:'English' Object has no attribute 'path'

**Content of configuration file** (if used & relevant): I am receiving this error while trying to run python -m rasa_nlu.train -c sample_configs/config_spacy.json
```json

```

**Comment by wrathagom** [comment]
Shot in the dark, did you already download and link the Spacy model? 

https://spacy.io/docs/usage/models

**Comment by seeni18** [comment]
Hi ,

I have downloaded model and linked it already.
It works fine when I import and execute as shown in link.

However when I try to.execute using rasa it throws that error.
Thanks
Srini

**Comment by tmbo** [comment]
This seems to be an issue with spacy 2.0 - currently that is not supported yet. #456

**Comment by wrathagom** [comment]
@seeni18 can you confirm you were trying to use spacy 2.0?

**Comment by seeni18** [comment]
How can I check the version? I am a new be to python world so..

Regards
Srinivas

On Sep 5, 2017 6:00 PM, "Caleb M. Keller" <notifications@github.com> wrote:

@seeni18 <https://github.com/seeni18> can you confirm you were trying to
use spacy 2.0?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/RasaHQ/rasa_nlu/issues/559#issuecomment-327160711>, or mute
the thread

**Comment by wrathagom** [comment]
What is the output of `pip show spacy`

**Comment by tmbo** [comment]
I'll close this for now - let me know if you were not using spacy 2.0

**Comment by csfyrakis** [comment]
Got the same issue with spacy 1.9


>   File "rasa_nlu/utils/spacy_utils.py", line 53, in create
    cls.ensure_proper_language_model(nlp)
  File "rasa_nlu/utils/spacy_utils.py", line 109, in ensure_proper_language_model
    if nlp.path is None:
AttributeError: 'English' object has no attribute 'path'
➜  rasa_nlu git:(master) ✗
➜  rasa_nlu git:(master) ✗
➜  rasa_nlu git:(master) ✗ pip show spacy
Name: spacy
Version: 1.9.0
Summary: Industrial-strength Natural Language Processing (NLP) with Python and Cython
Home-page: https://spacy.io
Author: Matthew Honnibal
Author-email: matt@explosion.ai
License: MIT
Location: /Users/44045241/anaconda2/lib/python2.7/site-packages
Requires: ftfy, requests, pathlib, plac, numpy, regex, dill, preshed, ujson, cymem, murmurhash, six, thinc, pip

**Comment by tmbo** [comment]
Didn't try 1.9 yet. Created a new issue #605

Is that using the `spacy_sklearn` default configuration?

**Comment by tmbo** [comment]
@csfyrakis just noticed that I have 1.9 installed myself. so it seems to be working. Can you explain a bit more detailed how to reproduce that issue?

**Comment by tmbo** [comment]
(best would be on issue #605 )
